apps/server/src/shared/domain/entity/coursegroup.entity.spec.ts

```diff
@@ -15,28 +14,32 @@ describe('CourseEntity', () => {
 	});
 
 	describe('constructor', () => {
-		it('should throw an error by empty constructor', () => {
+		it('should throw an error if constructor is called without arguments', () => {
 			// @ts-expect-error: Test case
 			const test = () => new CourseGroup();
+
 			expect(test).toThrow();
 		});
 
-		it('should create a course by passing required properties', () => {
+		it('should create a courseGroup when passing required properties', () => {
+			const name = 'someName';
 			const course = courseFactory.build();
-			course._id = new ObjectId();
-			const courseGroup = new CourseGroup({ course });
+
+			const courseGroup = new CourseGroup({ name, course });
+
 			expect(courseGroup instanceof CourseGroup).toEqual(true);
 		});
 	});
 
 	// TODO check if and how we need this
 	describe('getParent', () => {
 		it('should return the right id.', () => {
+			const name = 'someName';
 			const course = courseFactory.build();
-			course._id = new ObjectId();
-			const courseGroup = new CourseGroup({ course });
+			const courseGroup = new CourseGroup({ name, course });
```

Comment:
new CourseGroup > switch to factorys in this file?

Reply:
This is a test for the contructor. That's why it is newed here.